### PR TITLE
CIV-7603 add warnings to a11y test

### PR DESCRIPTION
**Pull request by laarmada**
### JIRA link ###

https://tools.hmcts.net/jira/browse/CIV-7603

### Change description ###

add coverage for warning in accessibility tests

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/test/a11y/a11y.test.ts [hunk 1]
```diff
@@ -65,13 +67,232 @@ function expectNoErrors(messages: PallyIssue[]): void {
...
+  const nonTextContent  = messages.filter(m => m.code.includes('WCAG2AA.Principle1.Guideline1_1'));
+  if (nonTextContent.length > 0) {
+    const errorsAsJson = `${JSON.stringify(nonTextContent, null, 2)}`;
+    fail(`There are non text content Guideline 1.1.1 issues: \n${errorsAsJson}\n`);
+  }
+
+}
+
+function expectNoAdaptableWarnings(messages: PallyIssue[]): void {
+  //const info  = messages.filter(m => m.code.includes('WCAG2AA.Principle1.Guideline1_3.1_3_1'));
```
Comment (laperla11) [inline src/test/a11y/a11y.test.ts:80]: Do you need the commented codes?

File: src/test/a11y/a11y.test.ts [hunk 2]
```diff
@@ -80,6 +301,13 @@ function testAccessibilityWithActions(url: string, actions: string[]): void {
 }
 
 function testAccessibility(url: string): void {
+  //we need to add actions for example:
```
Comment (laperla11) [inline src/test/a11y/a11y.test.ts:304]: Should we do it part of another task?

**Review by laperla11 — CHANGES_REQUESTED** [review]

File: src/test/a11y/a11y.test.ts (hunk 2 above)
Comment (laarmada) [reply src/test/a11y/a11y.test.ts:304]: exactly, only looking to make easy the next issue.

File: src/test/a11y/a11y.test.ts (hunk 1 above)
Comment (laarmada) [reply src/test/a11y/a11y.test.ts:80]: if we uncomment that lines we have accessibility issue so I will create an specific issue to fix each commented section.

**Pull request merged**